.github/workflows/ci.yml

```diff
@@ -93,13 +95,43 @@ jobs:
     - name: Run android integration tests
       run: |
         ./scripts/run-mocked-integration-tests-android.sh
+
+  build-macos-ios:
+    name: flutter build macos ios
+    runs-on: macos-12
+    env:
+      FLUTTER_VERSION: "3.0.3"
+    steps:
+    - name: Checkout 🛎️
+      uses: actions/checkout@v3
+      with:
+        persist-credentials: false
+    - name: Checkout submodules
+      run: git submodule update --init --recursive
+    - name: Create empty token file
+      run: echo 'const token = "";' > test_app/integration_tests/end_to_end/token.dart
+    - name: Set up Flutter version 🔧
+      uses: ./.github/actions/flutter-action
+      with:
+        flutter-version: $FLUTTER_VERSION
+    - name: Fetch dependencies
+      run: |
+        flutter pub get
+    - name: Run tests
+      run: |
+        flutter test -r expanded
+    - name: Run pana analyzer
+      run: |
+        ./scripts/score_analyzer.sh
```

Comment:
We are running both unit tests and pana on the linux build machine. I am not sure we need to run it here as well. And this goes for the Android build on Mac as well. What do you think @Bindu-Maharudrappa and @kbetl-dlb ?

Reply:
I agree with you. We can remove it from here I suppose.